Jakefile

```diff
@@ -17,14 +17,14 @@ task("cleanup", () => {
 
 desc("runs wslint on the built source");
 task("lint", ["build"], {async: true}, () => {
-  jake.exec([`cd ${DIST_DIR}; eslint .`], {
+  jake.exec([`cd ${DIST_DIR} & eslint .`], {
```

Comment:
I'm sorry? This runs `cd` in the background while running eslint in another directory. What were your intentions here?